Jenkinsfile

```diff
@@ -377,6 +377,46 @@ pipeline {
                     }
                 }
 
+                stage('Hip Implicitgemm layouts Release') {
+                    agent{ label rocmnode("vega") }
+                    environment{
+                        cmd = '''
+                            # check out MLIR
+                            dir=$(pwd)
+                            cd
+                            git clone -b miopen-dialect https://github.com/ROCmSoftwarePlatform/llvm-project-mlir.git
+
+                            # make build directory
+                            cd llvm-project-mlir && mkdir -p build && cd build
+
+                            # config MLIR on ROCm, with MIOpen dialect
+                            cmake -G Ninja ../llvm \
+                              -DLLVM_ENABLE_PROJECTS="mlir" \
+                              -DCMAKE_BUILD_TYPE=Release \
+                              -DBUILD_SHARED_LIBS=OFF \
+                              -DLLVM_BUILD_LLVM_DYLIB=OFF \
+                              -DLLVM_ENABLE_TERMINFO=OFF
+
+                            # build libMLIRMIOpen
+                            cmake --build . --target libMLIRMIOpen
```

Comment:
Also this should be installed and then the source and build directory should be deleted.

Reply:
This goes away together with the docker. A few lines above, it create and build MLIR in the home directory, not the mounted jenkins workspace directory.

Reply:
It should be installed into /usr/local. Ideally, we should do `make package` and install this in place of the llvm package provided by rocm. It seems mismatching across different llvm versions could cause problems. Although, updating the llvm version will need adjustments or fixing with clang-tidy.

Reply:
This cmake build will result in calling of a `add_custom_command` on the post build stage that put the built library under `/opt/rocm/lib`. 

> we should do make package and install this in place of the llvm package provided by rocm

So yes, as you mentioned, this is a different llvm version than the one provided by rocm. Since everything is zipped in this one target fat static library, I think it is better to make it separate from the rocm provided llvm.